projects/step-frontend/src/lib/modules/admin/components/my-account/my-account.component.html

```diff
@@ -1,77 +1,78 @@
 <mat-card>
-  <mat-card-header>
-    <mat-card-title>Credentials</mat-card-title>
+  <mat-card-header class="header-with-button">
+    <mat-card-title>User</mat-card-title>
+    <button type="button" mat-flat-button color="primary" (click)="changePwd()" [disabled]="!canChangePassword">
+      Change password
+    </button>
   </mat-card-header>
   <mat-card-content class="credentials">
-    <section>
+    <section class="text-field-section">
       <step-form-field alignLabelAddon="near">
         <step-label>Username</step-label>
         <step-label-addon>{{ user.username }}</step-label-addon>
       </step-form-field>
-      <button type="button" mat-flat-button color="primary" (click)="changePwd()" [disabled]="!canChangePassword">
-        Change password
-      </button>
     </section>
-    <section>
+    <section class="text-field-section">
       <step-form-field alignLabelAddon="near">
-        <step-label>Api key</step-label>
+        <step-label>Role</step-label>
+        <step-label-addon>{{ user.role }}</step-label-addon>
       </step-form-field>
-      <button
-        type="button"
-        mat-flat-button
-        color="primary"
-        (click)="invokeShowGenerateApiKeyDialog()"
-        [disabled]="!canGenerateApiKey"
-      >
-        Generate new Key
-      </button>
     </section>
-    <step-table [dataSource]="tokensSource">
-      <ng-container matColumnDef="label">
-        <th mat-header-cell *matHeaderCellDef>Label</th>
-        <td mat-cell *matCellDef="let element">
-          {{ element.label || element.id }}
-        </td>
-      </ng-container>
-      <ng-container matColumnDef="created">
-        <th mat-header-cell *matHeaderCellDef>Created</th>
-        <td mat-cell *matCellDef="let element">
-          {{ element.creationTime | date }}
-        </td>
-      </ng-container>
-      <ng-container matColumnDef="expired">
-        <th mat-header-cell *matHeaderCellDef>Expired</th>
-        <td mat-cell *matCellDef="let element">
-          {{ element.expiryTime | date }}
-        </td>
-      </ng-container>
-      <ng-container matColumnDef="actions">
-        <th class="actions" mat-header-cell *matHeaderCellDef></th>
-        <td mat-cell *matCellDef="let element">
-          <button mat-icon-button matTooltip="Revoke token" (click)="revokeAPIToken(element.id)">
-            <step-icon name="trash-2"></step-icon>
-          </button>
-        </td>
-      </ng-container>
-    </step-table>
   </mat-card-content>
 </mat-card>
 <mat-card>
-  <mat-card-header>
-    <mat-card-title>Role</mat-card-title>
+  <mat-card-header class="header-with-button">
+    <mat-card-title class="title-with-hint">
+      <div>Api key</div>
+      <step-description-hint [description]="apiKeyDescription"></step-description-hint>
```

Comment:
I know that this is not consequently followed in the App (I will put it up for discussion on Monday), but I prefer to have texts defined in the template file.